src/regtech_cleanup_api/routers/institution_cleanup.py

```diff
@@ -43,29 +43,33 @@ def delete_institution(request: Request, lei: str):
             detail=f"{lei} not valid test lei.",
         )
     else:
-        delete_domains = repo.delete_domains_by_lei(request.state.db_session, lei)
-        if not delete_domains:
-            logger.error(f"Domain(s) for LEI {lei} not deleted.")
+        return delete_helper(lei, request.state.db_session)
 
-        delete_sbl_type = repo.delete_sbl_type_by_lei(request.state.db_session, lei)
-        if not delete_sbl_type:
-            logger.error(f"sbl type(s) for LEI {lei} not deleted.")
 
-        res = repo.delete_institution(request.state.db_session, lei)
-        if not res:
+def delete_helper(lei: str, session: Session):
+    delete_domains = repo.delete_domains_by_lei(session, lei)
+    if not delete_domains:
+        logger.error(f"Domain(s) for LEI {lei} not deleted.")
```

Comment:
I'm guessing there are some test scenarios where there won't be email domains...like if you're testing that that user cannot get past that step in the process _without_ those domains. :) It doesn't seem like an `error` in that case. Maybe a `warn`?